### ISSUE TITLE
[DJANGO] Custom rules for Django migrations apply to non-Django project

### ISSUE THREAD

**Issue by frafra** [NINAnor/python-template#15]
https://github.com/NINAnor/python-template/blob/6c8cebe5d4b316c7b74c6192d5454b2de166c77c/pyproject.toml#L9

This code is in the main `pyproject.toml` file, shared across templates. There is no comment explaining what it is for, but I assume that is for Django project only, right?

**Comment by nicokant** [comment]
No, actually it's for the template itself. It contains some migrations file that needs to be ignored/treated differently when running pre-commit

**Comment by nicokant** [comment]
Yes, we use include to avoid copy-paste, but you cannot include just a part of a regular file

**Comment by frafra** [comment]
So this is a technical limitation of how copier as a whole works, and the only solution seems to copy-and-paste snippets of codes, which increases the burden on the maintainers of the template. I value more generating a correct project for the users of the template than making things more complicated for the maintainers of the template.

Is there any open discussion about this in the copier community?
